### DISCUSSION TITLE
Sponsoring/donating to the project?

### DISCUSSION THREAD

**Discussion by ajhenry** [OpenRCT2/OpenRCT2#17270] [Q&A] [answered]
Hey all!

I honestly love the work the maintainers and community is putting in and was wondering if there is a way to sponsor the project either through something like GitHub sponsors or even a donation link.

Thanks!

**Comment by janisozaur** [comment, marked as answer]
You can sponsor individual developers through GitHub sponsors:

https://github.com/sponsors/broxzier
https://github.com/sponsors/IntelOrca
https://github.com/sponsors/janisozaur
https://github.com/sponsors/gymnasiast

The project itself doesn't take donations.

**Reply by IntelOrca** [reply]
If you provide us with your discord user name as well, we can also provide you with the 'patron' role.
